generate_ctx.sh

```diff
@@ -1,6 +1,3 @@
 #!/bin/bash
+python3 tools/python/m2ctx.py
```

Comment:
If this is all the file is doing, I'd like to make the argument that `generate_ctx.sh` can be removed entirely and instead +x on `m2ctx.py` so it can be executed directly.

Reply:
Not sure if I want to move m2ctx.py outside of the `tools/python` folder, and having this makes it consistent with the other scripts as well.

Reply:
Thanks, agree with keeping consistent